style.css

```diff
@@ -0,0 +1,104 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-primary-text-1: #ff6a01;
+  --color-supporting-text-4: #fef5e0;
+  --color-icons-8: #dbd8d7;
+  --color-description-text-11: #fff;
+  --color-background-1: #1c1a19;
+  --color-nav-bar: #3c3a39;
+}
+
+.header {
+  width: 375px;
+  height: 48px;
+  background-color: var(--color-nav-bar, #3d3b39);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px 24px 2px 12px;
+}
+
+.header .logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  color: var(--color-icons-8);
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 375px;
+  height: 98vh;
+  gap: 12px;
+  padding: 120px 24px;
+  background-image: url(./assets/images/illustration-header-mobile.png);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-color: var(--color-background-1);
+}
+
+.hero .greeting {
+  position: relative;
+  width: 300px;
+  height: 124px;
+  font-family: 'Crete Round', serif;
+  color: var(--color-primary-text-1, #ff6b00);
+  font-size: 56px;
+  font-weight: 400px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  font-style: normal;
+  text-align: left;
+  margin-bottom: 0;
+}
+
+.hero .title {
+  display: flex;
+  align-items: center;
+  flex-grow: 0;
+  width: 327px;
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  color: var(--color-supporting-text-4);
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 32px;
+  font-style: normal;
+  margin-top: 10px;
+}
+
+.hero .paragraph {
+  width: 327px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  font-style: normal;
+  color: var(--color-description-text-11);
+  margin-top: 10px;
+}
+
+.social__media-icons .links {
+  display: flex;
+  width: 200px;
```

Comment:
- [ ] Kindly 🙏🏻 adhere to the project requirements properly, because as per the instructions your design should be responsive from `>=375px` to `<768px` screen size, but because of your `fixed` width as you gave here, your design responsiveness is restricted to only 375px screen-size, not more than 375px not will properly on less than 375px screen size. Please fix the issue and fulfil the below requirements properly. 

### Requirements

![image](https://github.com/DanielTagua/microverse-project-porfolio-1/assets/66476329/9250f276-d709-4526-9986-fa4ebb16b42b)

**Note: Use the 100% width for all container elements.**